pkg/server/routes.go

```diff
@@ -292,29 +291,30 @@ func (s *Server) routes(
 	}
 
 	if ok, _ := strconv.ParseBool(os.Getenv("DEBUG_ROUTES")); ok {
+		s.logger.Debug("debugging routes")
 		// useful for debugging route issues
 		_ = s.router.Walk(func(route *mux.Route, router *mux.Router, ancestors []*mux.Route) error {
 			pathTemplate, err := route.GetPathTemplate()
 			if err == nil {
-				fmt.Println("ROUTE:", pathTemplate)
+				s.logger.Debug("ROUTE: ", zap.String("ROUTE", pathTemplate))
 			}
 			pathRegexp, err := route.GetPathRegexp()
```

Comment:
NIT: Unrelated to this ticket but I normally see and expect "Regex", not "Regexp"

Reply:
yeah, I agree. Looks like this came from the `*mux.Router` library, so that's probably why they kept the log statement the same.